src/screens/login/login-validation.ts

```diff
@@ -31,3 +31,9 @@ export const isPhoneValid = (phone: string) => {
   const phoneRegex = /(\(?\d{2}\)?\s)?(\d{4,5}\d{4})/g;
   return phoneRegex.test(phone);
 };
+
+export const roleIsValid = (role: string) => {
```

Comment:
```suggestion
export const isRoleValid = (role: string) => {
```

A gente costuma considerar booleans com nomenclatura mais clara quando estão no formato de pergunta.